backend/app/services/runtime_work_kind_store.py

```
@@ -0,0 +1,214 @@
+    user_id: int,
+    project_ids: Optional[list[int]] = None,
+) -> list[DeviceWorkspaceResponse]:
+    """List active DeviceWorkspace Kinds for one user."""
+
+    rows = (
+        db.query(Kind)
+        .filter(
+            Kind.user_id == user_id,
+            Kind.kind == DEVICE_WORKSPACE_KIND,
+            Kind.namespace == DEVICE_WORKSPACE_NAMESPACE,
+            Kind.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)